pkg/slo-controller/noderesource/noderesource.go

```diff
@@ -195,6 +199,88 @@ func (r *NodeResourceReconciler) updateGPUNodeResource(node *corev1.Node, device
 	return err
 }
 
+func (r *NodeResourceReconciler) updateGPUDriverAndModel(node *corev1.Node, device *schedulingv1alpha1.Device) error {
+	if device == nil {
+		return nil
+	}
+
+	hasGPUDevice := false
+	for _, device := range device.Spec.Devices {
+		if device.Type != schedulingv1alpha1.GPU {
+			continue
+		}
+		hasGPUDevice = true
+	}
+
+	if !hasGPUDevice {
+		return nil
+	}
+
+	modelList, driverVersion, err := getGPUModelAndDriverInfo()
+	if err != nil {
+		klog.Errorf("failed to get GPU Model and driver info: %v", err)
```

Comment:
should add the  name of node name or device CRD object in the log.

Reply:
fixed